engine/apps/mobile_app/tasks.py

```diff
@@ -483,48 +483,46 @@ def conditionally_send_going_oncall_push_notifications_for_all_schedules() -> No
 
 # TODO: break down tasks.py into multiple files
 
-# Don't send notifications for shift swap requests that start more than 4 weeks in the future
-SSR_EARLIEST_NOTIFICATION_OFFSET = datetime.timedelta(weeks=4)
-
-# Once it's time to send out notifications, send them over the course of a week.
-# This is because users can be in multiple timezones / have different working hours configured,
-# so we can't just send all notifications at once, but need to wait for the users to be in their working hours.
-# Once a notification is sent to a user, they won't be notified again for the same shift swap request for a week.
-# After a week, the shift swap request won't be in the notification window anymore (see _get_shift_swap_requests_to_notify).
-SSR_NOTIFICATION_WINDOW = datetime.timedelta(weeks=1)
-
 
 @shared_dedicated_queue_retry_task()
 def notify_shift_swap_requests() -> None:
     """
     A periodic task that notifies users about shift swap requests.
     """
-    for shift_swap_request in _get_shift_swap_requests_to_notify(timezone.now()):
-        notify_shift_swap_request.delay(shift_swap_request.pk)
+    for shift_swap_request, timeout in _get_shift_swap_requests_to_notify(timezone.now()):
+        notify_shift_swap_request.delay(shift_swap_request.pk, timeout)
 
 
-def _get_shift_swap_requests_to_notify(now: datetime.datetime) -> list[ShiftSwapRequest]:
+def _get_shift_swap_requests_to_notify(now: datetime.datetime) -> list[tuple[ShiftSwapRequest, int]]:
     """
     Returns shifts swap requests that are open and are in the notification window.
     This method can return the same shift swap request multiple times while it's in the notification window,
     but users are only notified once per shift swap request (see _mark_shift_swap_request_notified_for_user).
     """
+    # avoid circular import
+    from apps.schedules.tasks.shift_swaps.slack_followups import FOLLOWUP_OFFSETS
```

Comment:
consider moving `FOLLOWUP_OFFSETS` as maybe an attribute on the `ShiftSwapRequest` model, as it's no longer specific to just the Slack celery task.

Reply:
Good catch, addressed in https://github.com/grafana/oncall/pull/2805/commits/625095b27fd329b1c7785dca97f0235f54342a60